Makefile

```diff
@@ -27,15 +31,15 @@ clean_test_deps:
 ${TEST_CRDS}/gitrepositories.yaml:
 	mkdir -p ${TEST_CRDS}
 	curl -s --fail https://raw.githubusercontent.com/fluxcd/source-controller/${SOURCE_VER}/config/crd/bases/source.toolkit.fluxcd.io_gitrepositories.yaml \
-		-o ${TEST_CRDS}/gitrepositories.yaml
+		-o ${TEST_CRDS}/gitrepositories_${SOURCE_VER}.yaml
```

Comment:
Changing this requires a change of the target, as otherwise the `make` behavior stops working resulting in the file being downloaded all the time.